src/platform_impl/macos/window.rs

```diff
@@ -134,6 +136,7 @@ unsafe fn create_view(
   })
 }
 
+// TODO: Modify this
```

Comment:
why modify this?

Reply:
Honestly... I have no clue. I worked on this a long time ago